src/js/modules/articles/reducer.js

```diff
@@ -25,7 +23,17 @@ const reducer = (state = { ...initialState }, action) => {
         ...state,
         isFetching: false,
         isFetched: true,
-        response: action.payload,
+        /* Placing state.articles as the initial value will for some reason
+         * inhibit components' updating with the articles state, which is why
+         * the spread operator is required.
+         */
+        articles: {
+          ...state.articles,
+          ...action.payload.reduce((acc, article) => {
```

Comment:
Cause you're fucking with operator precedence at that point, sorta like `f(++a)`